bedrock/newsletter/views.py

```diff
@@ -635,13 +635,18 @@ def newsletter_subscribe(request):
             if not kwargs.get("source_url") and request.headers.get("Referer"):
                 kwargs["source_url"] = request.headers["Referer"]
 
+            # Convert data["newsletters"] to a comma separated string.
+            newsletters = data["newsletters"]
+            if isinstance(newsletters, list):
+                newsletters = ",".join(newsletters)
```

Comment:
Nit: Do want this to be `", "` so that we get `"foo, bar"`, not `"foo,bar"`?

Reply:
This is only how we send it to basket, which looked like it preferred no spaces. It may take both, but this isn't displayed to the user so I chose just CSV format.